stubs/Symfony/Component/OptionsResolver/Options.stub

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Symfony\Component\OptionsResolver;
```

Comment:
Needs to be registered in `stubFiles` in `extension.neon`.

Also since you're making `Options` class generic, you need to add it to `parameters.featureToggles.skipCheckGenericClasses`, similarly to here: https://github.com/phpstan/phpstan-src/blob/74228e6f930c9a0e6ca9461557a6d40b92cf4e91/conf/config.neon#L23-L33 (this list will be purged with the next major version).